docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+services:
+  minio:
+    container_name: minio_container
+    network_mode: host
+    image: quay.io/minio/minio
+    command: server /data --console-address ":9001"
+    environment:
+      - "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE"
+      - "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+  miniocreatebuckets:
+    image: minio/mc
+    network_mode: host
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      until (mc alias set hummock-minio http://127.0.0.1:9000 AKIAIOSFODNN7EXAMPLE wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY) do echo '...waiting...' && sleep 1; done;
+      mc admin user add hummock-minio/ hummock 12345678;
+      mc admin policy set hummock-minio/ readwrite user=hummock;
+      mc mb hummock-minio/hummock001;
+      exit 0;
+      "
+  metanode:
+    container_name: metanode_container
+    network_mode: host
+    image: ghcr.io/singularity-data/risingwave-backend:latest
+    command: /bin/bash -c /risingwave/bin/meta-node --host 127.0.0.1:5690 --dashboard-host 127.0.0.1:5691 --prometheus-host 127.0.0.1:1250
+    depends_on:
+      - miniocreatebuckets
+  computenode:
+    container_name: computenode_container
+    network_mode: host
+    image: ghcr.io/singularity-data/risingwave-backend:latest
+    command: /bin/bash -c /risingwave/bin/compute-node --config-path /risingwave/config/risingwave.toml --host 127.0.0.1:5688 --prometheus-listener-addr 127.0.0.1:1222 --metrics-level 1 --state-store hummock+minio://hummock:12345678@127.0.0.1:9000/hummock001 --meta-address http://127.0.0.1:5690
+    depends_on:
+      - metanode
+  frontend:
+    container_name: frontend_container
+    network_mode: host
+    image: ghcr.io/singularity-data/risingwave-frontend:latest
+    command: /bin/bash -c /risingwave/bin/frontend-v2 --host 127.0.0.1:4566 --meta-addr http://127.0.0.1:5690
+    depends_on:
+      - metanode
```

Comment:
also depends on computenode

Reply:
committed. but i don't think it matters. 
`depends_on` only waits for the other container to be up, but not for the process it is running to start. 
If we need to guarantee the start order of the services, we may need other mechanism.
Based on my test, the order does not affect. It always work. Except `miniocreatebuckets`.